doc/general/installation/installation.md

```diff
@@ -3,56 +3,138 @@ slug: /poscreators/middleware-doc/general/installation
-| `-connectionretry`             | Sets the number of trials to download the configuration in the static configuration (`fiskaltrust.exe.config`) |
-| `-proxy`                       | Sets the proxy server to be used to connect to internet in the static configuration (`fiskaltrust.exe.config`) |
+The received zip-compressed folders need to be unzipped and can be moved or renamed if necessary.
 
+The folder with the downloaded and unzipped launcher contains 
+
+- the launcher `fiskaltrust.exe`, 
+- three pre-configured `.cmd` command files,
+- a file for the static configuration of the service named `fiskaltrust.exe.config`,  and 
+- the fiskaltrust.Middleware service represented by the `.dll` files.
+
+## Windows, Linux & MacOS
+
+### Launcher configuration
+
+In the fiskaltrust.Portal, the components of the fiskaltrust.Middleware have been configured via Cashbox configuration for the environment where the Middleware should be operated, e.g. which database should be used to store the receipt data. 
+
+The downloaded Launcher can now be optionally adapted for the local machine; e.g. setting the target folder for the data storage of the service, or setting the service name. This can be done by either using call parameters when executing the `fiskaltrust.exe`, or by directly modifying the  `fiskaltrust.exe.config`  by adding the appropriate key-value pairs in the section `appSettings`.
+
+The following call parameters are available with the launcher `fiskaltrust.exe`:
+
+| **Parameter**                  | **Description**                                              | Overwrites the values in the static configuration `fiskaltrust.exe.config` | AT              | DE        | FR              |
+| ------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | --------------- | --------- | --------------- |
+| `-cashboxid`                   | Sets the CashBoxId. The value is a GUID in format `00000000-0000-0000-0000-000000000000`. | yes                                                          | supported       | supported | supported       |
+| `-accesstoken`                 | Sets the AccessToken for online communication                | yes                                                          | supported       | supported | supported       |
+| `-useoffline`                  | Sets the offline mode. The value is a boolean: true \| false | yes                                                          | supported       | supported | supported       |
+| `-test`                        | Executing as command line program. Basic information is provided in the console. Should be indicated as last parameter, if it is set in connection with others. | no                                                           | supported       | supported | supported       |
```

Comment:
the test parameter does not need to be the last parameter anymore.

Reply:
cool, thank you a lot for that!